### PR TITLE
Bump docker tags

### PR DESCRIPTION
**Pull request by jgrund**
Bump docker tags to 5.1.1-dev, which is not in the registry.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1562)

### CODE REVIEW

File: docker/base.dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN yum update -y \
   && yum install -y python python-pip python-devel postgresql openssl gcc-c++ \
   && pip install -r requirements.txt \
   && yum autoremove -y gcc-c++ python-pip python-devel \
+  && rm -rf /root/.cache/pip \
```
Comment (ip1981) [inline docker/base.dockerfile]: Nit: `/root/.cache` or even `/root/` might be good :)

Comment (jgrund) [reply docker/base.dockerfile]: Not sure I'm comfortable removing everything under root. Removing the pip cache here saves about 22MB.

**Comment by jgrund** [comment]
Confirmed this works with a docker stack deploy.

Images that are loaded and not in the registry:

Screen Shot 2020-02-07 at 9 07 15 AM

Stack running from the local images.

Screen Shot 2020-02-07 at 9 07 01 AM

**Comment by johnsonw** [comment]
One thing you are probably missing is `iml-wasm-components` inside the nginx.dockerfile

**Comment by jgrund** [comment]
> One thing you are probably missing is `iml-wasm-components` inside the nginx.dockerfile

I think it's fine to keep that as is. I don't see us furthering development there.

**Comment by johnsonw** [comment]
Ok, everything else looks good. I'll +1

**Pull request merged**